pandas/tests/io/test_common.py

```diff
@@ -89,6 +107,68 @@ def test_iterator(self):
         tm.assert_frame_equal(first, expected.iloc[[0]])
         tm.assert_frame_equal(concat(it), expected.iloc[1:])
 
+    @pytest.mark.parametrize('reader, module, path', [
```

Comment:
yeah, I think this is fine actually, though #16292 would need to change (you can rebase on top maybe)?